src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.cpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
+
+#include <pup.h>  // IWYU pragma: keep
+
+#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "Utilities/MakeWithValue.hpp"
+
+namespace Elasticity {
+namespace ConstitutiveRelations {
+
+template <size_t Dim>
+IsotropicHomogeneous<Dim>::IsotropicHomogeneous(double incompressibility,
+                                                double rigidity) noexcept
+    : incompressibility_(incompressibility), rigidity_(rigidity) {}
+
+template <>
+template <typename DataType, typename Frame>
+tnsr::II<DataType, 3, Frame> IsotropicHomogeneous<3>::stress(
+    const tnsr::ii<DataType, 3, Frame>& strain,
+    const tnsr::I<DataType, 3, Frame>& /*x*/) const noexcept {
+  auto result = make_with_value<tnsr::II<DataType, 3, Frame>>(strain, 0.);
+  for (size_t i = 0; i < 3; i++) {
+    for (size_t j = 0; j <= i; j++) {
+      result.get(i, j) = 2. * rigidity_ * strain.get(i, j);
+    }
+  }
+  auto trace_term = make_with_value<DataType>(strain, 0.);
+  for (size_t i = 0; i < 3; i++) {
+    trace_term += strain.get(i, i);
```

Comment:
You can probably use an existing trace() function here

Reply:
I think those require passing in a metric